src/Umbraco.Web.Common/AspNetCore/AspNetCoreCookieManager.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using Microsoft.AspNetCore.Http;
+using Umbraco.Core.Cookie;
```

Comment:
I had a task to cleanup this namespace since it shouldn't exist like that, just not sure where those notes went. There's a few of new namespaces in Core that have 'one' item which shouldn't exist and needs to be better organized. These are mostly to do with 'web' things. Anyways, i'll create/find the task for that or just do it after review.